### ISSUE TITLE
Android tv crashing with certain addons

### ISSUE THREAD

**Issue by Camzilla78** [Stremio/stremio-bugs#959]
Streams crash mid stream with addons easynews+ and Shuvam Gdrive. This seems to be an android tv issue as the same stream works fine on android mobile with same exact file and wifi connection. Tested multiple times with the same results in crashing. more users are using these addons so a fix to this work be awesome. A 4k file size of 5gb will crash with either addon at 8 minutes 50 seconds everytime.
Any show or movie with easynews+ or gdrive addons.
I tested From s3 e4 but its every stream
A stream will play for a little then crash.
Exo will try to switch to libvlc but crash to links 
 - Device: Firecube 3rd gen and Firecube 2nd gen.
 - OS: Android tv fire os 7.6.8.1
 - Version 6540163.1 Using 1.6.12 latest android tv apk

**Comment by Camzilla78** [comment]
Any insight into this would be highly appreciated as it's a frustrating issue to try to understand. Quite  a few users are having the same problems. Thanks in advance!!

**Comment by TRtomasz** [comment]
We tried the stream you reported and it was issue with the addon/video file as it was sending end of stream signal and thats why it was exiting. Perhaps it is doing this for more files

**Comment by Camzilla78** [comment]
You are correct, that was just an example. It does this for every stream eventually with these addons
